src/server/decommission-service/routes.js

```diff
@@ -0,0 +1,47 @@
+import { authScope } from '~/src/server/common/helpers/auth/auth-scope.js'
+import { scopes } from '~/src/server/common/constants/scopes.js'
+import { decommissionFormController } from '~/src/server/decommission-service/controllers/decommission-form.js'
+import { decommissionServiceController } from '~/src/server/decommission-service/controllers/decommission-service.js'
+import { summaryController } from '~/src/server/decommission-service/controllers/summary.js'
+import { provideSubNavigation } from '~/src/server/admin/helpers/provide-sub-navigation.js'
+
+const adminScope = authScope([`+${scopes.admin}`])
```

Comment:
with this folder going into the admin section. The entire section is wrapped in admin scope. So you will get this for free